diff --color -rupN linux-6.12.8/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c linux-6.12.8-patched/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c
--- linux-6.12.8/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c	2025-01-02 01:34:26.000000000 -0800
+++ linux-6.12.8-patched/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c	2025-01-06 21:26:52.634163560 -0800
@@ -3478,6 +3478,8 @@ static void update_connector_ext_caps(st
 		caps->aux_min_input_signal = 0;
 		caps->aux_max_input_signal = 512;
 	}
+
+	caps->min_input_signal = 0;
 }
 
 void amdgpu_dm_update_connector_after_detect(
