diff --color -rupN linux-6.12.8/drivers/bluetooth/btusb.c linux-6.12.8-patched/drivers/bluetooth/btusb.c
--- linux-6.12.8/drivers/bluetooth/btusb.c	2025-01-02 01:34:26.000000000 -0800
+++ linux-6.12.8-patched/drivers/bluetooth/btusb.c	2025-01-06 21:29:50.617708760 -0800
@@ -371,6 +371,8 @@ static const struct usb_device_id quirks
 	/* QCA WCN785x chipset */
 	{ USB_DEVICE(0x0cf3, 0xe700), .driver_info = BTUSB_QCA_WCN6855 |
 						     BTUSB_WIDEBAND_SPEECH },
+	{ USB_DEVICE(0x2c7c, 0x0130), .driver_info = BTUSB_QCA_WCN6855 |
+						     BTUSB_WIDEBAND_SPEECH },
 
 	/* Broadcom BCM2035 */
 	{ USB_DEVICE(0x0a5c, 0x2009), .driver_info = BTUSB_BCM92035 },
